### DISCUSSION TITLE
Can't configure s3 source

### DISCUSSION THREAD

**Discussion by rafaelrccs** [airbytehq/airbyte#30173] [Connector Questions]
Hi Everyone,

I'm trying to configure a s3 source but no matter if it is an s3 bucket, a local minio bucket or a bucket on digital ocean, I always receive the same error message when testing source:

"""Configuration check failed
The check connection failed because of an internal error

Learn more
Internal message: scheduledEventId=6, startedEventId=7, activityType='RunWithJobOutput', activityId='2de04419-2822-3144-908f-8891ae19c2e2', identity='1@1f30b41578d8', retryState=RETRY_STATE_MAXIMUM_ATTEMPTS_REACHED
Failure origin: source"""
Logs area attached.
[check_connection_source-failure-abab1bb7-2ba0-4515-94b6-4dde9652b23d.txt](https://github.com/airbytehq/airbyte/files/12527702/check_connection_source-failure-abab1bb7-2ba0-4515-94b6-4dde9652b23d.txt)

Can someone help on this issue?

[a comment by an automated account is not shown]

**Comment by lucasfcnunes** [comment]
:(
